service/src/main/java/bio/terra/workspace/service/workspace/flight/WorkspaceFlightMapKeys.java

```diff
@@ -22,7 +22,7 @@ private WorkspaceFlightMapKeys() {}
     private ControlledResourceKeys() {}
 
     public static final String CREATION_PARAMETERS = "creationParameters";
-    public static final String PRIVATE_RESOURCE_IAM_ROLES = "iamRoles";
+    public static final String PRIVATE_RESOURCE_IAM_ROLE = "privateResourceIamRole";
```

Comment:
Yes, this is a breaking change in many ways: it rejiggers what the Flight stores in the map. So even if I fixed this, it would still break. I _think_ this is OK for this moment, but as we have real production users, we will have to pick a compatibility approach.

I'll add a ticket for a design for that.